### PR TITLE
New package: wally-cli-2.0.0

### PR DESCRIPTION
**Pull request by waynevanson**

### CODE REVIEW

**Comment by Piraty** [comment]
[NOMERGE] until upstream settles on a license. https://github.com/zsa/wally-cli/issues/2

**Comment by waynevanson** [comment]
https://github.com/zsa/wally-cli/issues/2 has been closed.

File: srcpkgs/wally-cli/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'wally-cli'
+pkgname=wally-cli
+version=2.0.0
+revision=1
+archs="x86_64"
```
Comment (Piraty) [inline srcpkgs/wally-cli/template:5]: why is this limited to x86_64 (glibc)?

File: srcpkgs/wally-cli/files/50-wally.rules [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Teensy rules for the Ergodox EZ
```
Comment (Piraty) [inline srcpkgs/wally-cli/files/50-wally.rules:1]: did you come up with them?

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/wally-cli/files/50-wally.rules (hunk 2 above)
Comment (waynevanson) [reply srcpkgs/wally-cli/files/50-wally.rules:1]: I got them from https://github.com/zsa/wally/wiki/Linux-install#2-create-a-udev-rule-file

I've written this in the file at `files/50-wally.rules`

File: srcpkgs/wally-cli/template (hunk 1 above)
Comment (waynevanson) [reply srcpkgs/wally-cli/template:5]: I would've left it here for testing purposes when starting the file. I will update this to what works.

I cannot get these to cross compile and I've added what I can test

Comment (Piraty) [reply srcpkgs/wally-cli/template]: after readin the cross build log,
```
aarch64-linux-musl-pkg-config: exec: "aarch64-linux-musl-pkg-config": executable file not found in $PATH
```
it's obvious you have to  `hostmakedepends="pkg-config"`
this makes it cross compile.
please remove the arch restriction if it's not necessary

**Comment by Piraty** [comment]
I applied my suggestions and merged, i hope that's ok with you ;)

**Pull request closed without merging**